scripts/apps/authoring-react/data-layer.ts

```diff
@@ -122,6 +124,28 @@ function getArticleContentProfile<T>(item: IArticle, fieldsAdapter: IFieldsAdapt
             }
         }
 
+        // TODO: write an upgrade script and remove hardcoding
+        // after angular based authoring is removed from the codebase
+        if (item.type === 'picture' || item.type === 'audio' || item.type === 'video') {
+            const fieldConfig: IEditor3Config = {
+                editorFormat: [],
+                minLength: fakeScope.schema?.minlength,
+                maxLength: fakeScope.schema?.maxlength,
+                cleanPastedHtml: fakeScope.editor?.cleanPastedHTML,
+                singleLine: false,
+                disallowedCharacters: [],
+            };
+
+            const description_text: IAuthoringFieldV2 = {
+                id: DESCRIPTION_TEXT_FIELD_ID,
+                name: gettext('Description'),
+                fieldType: 'editor3',
+                fieldConfig,
+            };
+
+            contentFields = contentFields.set(description_text.id, description_text);
```

Comment:
looks like config is defined twice, here and in the adapter. Which one gets used in the end? I'd think that the adapter one., Which means that maybe you should be able to do

```
contentFields = contentFields.set(description_text.id, null);
```